vibe_code_apps_22/ml/eplus_residual_decomposition.py

```
@@ -0,0 +1,131 @@
+        "n": int(len(df)),
+        "mean_obs_kw": float(df[obs_col].mean()),
+        "mean_sim_kw": float(df[sim_col].mean()),
+        "overnight_00_05_mean_resid_kw": float(overnight["resid"].mean()) if len(overnight) else None,
+        "daytime_07_15_mean_resid_kw": float(daytime["resid"].mean()) if len(daytime) else None,
+        "weekend_mean_sim_kw": float(weekend[sim_col].mean()) if len(weekend) else None,
+        "weekend_mean_obs_kw": float(weekend[obs_col].mean()) if len(weekend) else None,
+        "weekday_peak_obs_kw": float(weekday[obs_col].max()) if len(weekday) else None,
+        "weekday_peak_sim_kw": float(weekday[sim_col].max()) if len(weekday) else None,
+        "structural_narrative": [
+            "Missing persistent overnight baseload / pump / fan / DOAS electricity "
+            "(model underpredicts hours 00–05).",
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)